docs/Project.toml

```diff
@@ -1,7 +1,6 @@
 [deps]
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-DocumenterCitations = "daee34ce-89f3-4625-b898-19384cb65244"
```

Comment:
probably removing DocumenterCitations will make docs built to fail

Reply:
I see the problem probably is that you "upgraded" Documenter to v0.28 in your fork and then DocumenterCitations can't handle it due to compat restriction here --> https://github.com/ali-ramadhan/DocumenterCitations.jl/blob/5ef4fbf4b71cc04941ad5258adea666c046b73da/Project.toml

So perhaps just add back the `-DEV` in the Documenter version for these experiments?

Reply:
Should I have changed the version on my fork to 0.27? I'm confused what the right process is here

Reply:
That would also worked!! :)

Reply:
Do that. Then no changes in DocumenterCitation are needed